From 180ed0eb6e8bf0605116596836ab5cd8376387e5 Mon Sep 17 00:00:00 2001
From: Zeno Albisser <zeno@webkit.org>
Date: Thu, 23 May 2013 11:12:26 +0200
Subject: [PATCH] remove render_widget_host_view_gtk and web_contents_view_gtk

---
 content/content_browser.gypi | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/content/content_browser.gypi b/content/content_browser.gypi
index 92798b9..9085fc9 100644
--- a/content/content_browser.gypi
+++ b/content/content_browser.gypi
@@ -832,8 +832,6 @@
     'browser/renderer_host/render_widget_host_view_aura.h',
     'browser/renderer_host/render_widget_host_view_base.cc',
     'browser/renderer_host/render_widget_host_view_base.h',
-    'browser/renderer_host/render_widget_host_view_gtk.cc',
-    'browser/renderer_host/render_widget_host_view_gtk.h',
     'browser/renderer_host/render_widget_host_view_guest.cc',
     'browser/renderer_host/render_widget_host_view_guest.h',
     'browser/renderer_host/render_widget_host_view_mac_editcommand_helper.h',
-- 
1.8.1.2

