From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Zoltan Arvai <zarvai@inf.u-szeged.hu>
Date: Mon, 24 Feb 2014 07:29:32 -0800
Subject: LIBVPX: Add path for QtWebEngine build in Windows specific actions

---
 libvpx.gyp |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/libvpx.gyp b/libvpx.gyp
index 392ccde..7f1a18d 100644
--- a/libvpx.gyp
+++ b/libvpx.gyp
@@ -402,6 +402,7 @@
                 '-s', '<(PRODUCT_DIR)/obj/libvpx_asm_offsets_vp8/vp8_asm_enc_offsets.obj',
                 '-s', '<(ninja_obj_dir)/encoder/libvpx_asm_offsets_vp8.vp8_asm_enc_offsets.obj',
                 '-s', '<(PRODUCT_DIR)/obj/Source/WebKit/chromium/third_party/libvpx/<(libvpx_source)/vp8/encoder/libvpx_asm_offsets_vp8.vp8_asm_enc_offsets.obj',
+                '-s', '<(qtwe_chromium_obj_dir)/third_party/libvpx/<(libvpx_source)/vp8/encoder/libvpx_asm_offsets_vp8.vp8_asm_enc_offsets.obj',
               ],
               'process_output_as_sources': 1,
               'msvs_cygwin_shell': 1,
@@ -500,6 +501,7 @@
                 '-s', '<(PRODUCT_DIR)/obj/libvpx_asm_offsets_vpx_scale/vpx_scale_asm_offsets.obj',
                 '-s', '<(ninja_obj_dir)/encoder/libvpx_asm_offsets_vpx_scale.vpx_scale_asm_offsets.obj',
                 '-s', '<(PRODUCT_DIR)/obj/Source/WebKit/chromium/third_party/libvpx/<(libvpx_source)/vpx_scale/libvpx_asm_offsets_vpx_scale.vpx_scale_asm_offsets.obj',
+                '-s', '<(qtwe_chromium_obj_dir)/third_party/libvpx/<(libvpx_source)/vpx_scale/libvpx_asm_offsets_vpx_scale.vpx_scale_asm_offsets.obj',
               ],
               'process_output_as_sources': 1,
               'msvs_cygwin_shell': 1,
