From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jocelyn Turcotte <jocelyn.turcotte@digia.com>
Date: Tue, 11 Jun 2013 15:44:26 +0200
Subject: Add WebEngineContext to RunLoop's friends.

---
 base/run_loop.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/base/run_loop.h b/base/run_loop.h
index 0dce634..055b1b8 100644
--- a/base/run_loop.h
+++ b/base/run_loop.h
@@ -10,6 +10,8 @@
 #include "base/memory/weak_ptr.h"
 #include "base/message_loop/message_loop.h"
 
+class WebEngineContext;
+
 namespace base {
 #if defined(OS_ANDROID)
 class MessagePumpForUI;
@@ -76,6 +78,7 @@ class BASE_EXPORT RunLoop {
 
  private:
   friend class MessageLoop;
+  friend class ::WebEngineContext;
 #if defined(OS_ANDROID)
   // Android doesn't support the blocking MessageLoop::Run, so it calls
   // BeforeRun and AfterRun directly.
